skills_DS/scripts/seed_database.py

```diff
@@ -0,0 +1,69 @@
iramichi', 'Cobourg', 'Dorval', 'Oak Bay', 'Amos', 'Spruce Grove', 'Pitt Meadows', 'Fort St. John', 'Deux-Montagnes', 'Saint-Augustin-de-Desmaures', 'Conception Bay South', 'Saint-Lazare', 'Sainte-Catherine', "L'Assomption", 'Edmundston', "L'Ancienne-Lorette", 'Cochrane', 'Lutes Mountain', 'Sainte-Catherine', 'Salmon Arm', 'Rayside-Balfour', 'Petawawa', 'Fort Erie', 'Candiac', 'Yellowknife', 'Collingwood', 'Mont-Saint-Hilaire', 'Camrose', 'Yorkton', 'Saint-Basile-le-Grand', 'Leduc', 'Pembroke', 'Magog', 'Greater Napanee', 'Kenora', 'le Plateau', 'Fort Saskatchewan', 'Okotoks', 'Gaspe', 'Matane', 'Summerside', 'Prince Rupert', 'Swift Current', 'Colwood', 'Rosemere', 'Williams Lake', 'Saint-Lin-Laurentides', 'Simcoe', 'Quesnel', 'Bay Roberts', 'Mont-Laurier', 'Portage la Prairie', 'Dolbeau-Mistassini', 'Powell River', 'Brooks', 'Bathurst', 'Kentville', 'Elliot Lake', 'Thompson', 'Aldergrove', 'Canmore', 'Hawkesbury', 'North Perth', 'Grand Falls-Windsor', 'Parksville', 'Beauharnois', 'Ingersoll', 'Montmagny', 'Cold Lake', 'Sainte-Marie', 'Uxbridge', 'Fall River', 'Amherstburg', 'Sainte-Marthe-sur-le-Lac', 'Wetaskiwin', 'Pincourt', 'Paris', 'Becancour', 'North Saanich', 'Cowansville', 'Dawson Creek', 'Sainte-Anne-des-Plaines', 'Lambton Shores', 'Sainte-Adele', 'Les Coteaux', 'Whistler', 'Smiths Falls', 'Sylvan Lake', 'Temiskaming Shores', 'Cantley', 'Rawdon', 'Labrador City', 'Angus', 'Prevost', 'Mercier', 'Lacombe', 'Strathmore', 'Hanceville', 'South Huron', 'Bells Corners', "L'Ile-Perrot", 'Estevan', 'Hinton', "Notre-Dame-de-l'Ile-Perrot", 'High River', 'Nelson', 'Carleton Place', 'Trail', 'Selkirk', 'Lorraine', 'Steinbach', 'Arnprior', 'Lachute', 'Val-des-Monts', 'La Tuque', 'Weyburn', 'Amherst', 'Dorchester', 'Roberval', 'Saint-Raymond', 'Huron East', 'Kapuskasing', 'Saint-Sauveur', 'Oromocto', 'Kitimat', 'La Malbaie', 'Stony Plain', 'Mont-Tremblant', 'Binbrook', 'Whitecourt', 'Lavaltrie', 'Ladysmith', 'Pont-Rouge', 'Castlegar', 'Otterburn Park', 'Dauphin', 'Bois-des-Filion', 'Winkler', 'Concord', 'Bracebridge', 'Notre-Dame-des-Prairies', 'Dryden', 'Taber', 'Morinville', 'Silver Berry', 'Saint-Felicien', 'Saint-Hippolyte', 'Bridgewater', 'Goderich', 'Wild Rose', 'Renfrew', 'Edson', 'Fort Frances', 'Chandler', 'Kirkland Lake', 'Slave Lake', 'Happy Valley-Goose Bay', 'Wasaga Beach', 'Chibougamau', 'Revelstoke', 'Aylmer', 'Pont Rouge', 'Camlachie', 'Banff', 'Yarmouth', 'Carignan', 'Campbellton', 'Beaumont', 'Innisfil', 'Delson', 'Sydney Mines', 'Baie-Saint-Paul', 'Hanover', 'La Sarre', 'Asbestos', 'Merritt', 'Bluewater', 'Coaticook', 'Hampstead', 'Olds', 'Greenwood', 'Gibsons', 'Carbonear', 'Saint-Zotique', 'Kincardine', 'Plessisville', 'Brownsburg-Chatham', 'Ponoka', 'Mont-Joli', 'Rocky Mountain House', 'Kimberley', 'Perth', 'Parry Sound', 'Pointe-Calumet', 'Morden', 'Coaldale', 'Summerland', 'Stephenville', 'Amqui', 'Beauceville', 'Saint-Joseph-du-Lac', 'Sooke', 'Iqaluit', 'Drayton Valley', 'Farnham', 'The Pas', 'Bromont', 'Saint-Felix-de-Valois']
```

Comment:
this seems like a lot of data for a constant. I would rename this to LOCATION or CONSTANTS_LOCATION but this is also fine.